ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -116,43 +165,124 @@ private void collectData(File[] replicas, ArrayList<String> replicasList, Map<St
+   *                                         status of every blob in the index
+   * @param replicaCount total replica count
+   * @param includeAcceptableInconsistentBlobs {@code true} if acceptable inconsistent blobs needs to be considered. {@code false}
+   *                                                       otherwise
+   * @param realInconsistentBlobs {@link List} of real inconsistent blobIds
+   */
+  private void populateOutput(AtomicLong totalKeysProcessed, Map<String, BlobStatus> blobIdToStatusMap,
+      int replicaCount, boolean includeAcceptableInconsistentBlobs, List<String> realInconsistentBlobs) {
     logger.info("Total keys processed " + totalKeysProcessed.get());
     logger.info("\nTotal Blobs Found " + blobIdToStatusMap.size());
+    long earliestRealInconsistentBlobTimeMs = Long.MAX_VALUE;
+    long latestOpTimeMs = Long.MIN_VALUE;
     long inconsistentBlobs = 0;
-    long realInconsistentBlobs = 0;
+    AtomicLong inconsistentDueToReplicationCount = new AtomicLong(0);
     long acceptableInconsistentBlobs = 0;
+    List<String> potentialInconsistentBlobs = new ArrayList<>();
     for (String blobId : blobIdToStatusMap.keySet()) {
       BlobStatus consistencyBlobResult = blobIdToStatusMap.get(blobId);
+      latestOpTimeMs = Math.max(latestOpTimeMs, consistencyBlobResult.getOpTime());
       // valid blobs : count of available replicas = total replica count or count of deleted replicas = total replica count
       // acceptable inconsistent blobs : count of deleted + count of unavailable = total replica count
       // rest are all inconsistent blobs
-      boolean isValid = consistencyBlobResult.getAvailable().size() == replicaCount
-          || consistencyBlobResult.getDeletedOrExpired().size() == replicaCount;
+      boolean isValid = consistencyBlobResult.getAvailableList().size() == replicaCount
+          || consistencyBlobResult.getDeletedOrExpiredList().size() == replicaCount;
       if (!isValid) {
         inconsistentBlobs++;
-        if ((consistencyBlobResult.getDeletedOrExpired().size() + consistencyBlobResult.getUnavailableList().size()
+        if ((consistencyBlobResult.getDeletedOrExpiredList().size() + consistencyBlobResult.getUnavailableList().size()
             == replicaCount)) {
           if (includeAcceptableInconsistentBlobs) {
             logger.info("Partially deleted (acceptable inconsistency) blob " + blobId + " isDeletedOrExpired "
-                + consistencyBlobResult.getIsDeletedOrExpired() + "\n" + consistencyBlobResult);
+                + consistencyBlobResult.isDeletedOrExpired() + "\n" + consistencyBlobResult);
           }
           acceptableInconsistentBlobs++;
         } else {
-          realInconsistentBlobs++;
-          logger.info(
-              "Inconsistent Blob : " + blobId + " isDeletedOrExpired " + consistencyBlobResult.getIsDeletedOrExpired()
-                  + "\n" + consistencyBlobResult);
+          potentialInconsistentBlobs.add(blobId);
```

Comment:
Have fixed it.
Total Inconsistent = Acceptable + Replication Lag + real inconsistent